docs: rbac in dashboard (#703)
mintlify/docs: advanced/dashboard/roles.mdx

---
title: "Roles"

> description: "Provide members of your team differing levels of access to the dashboard."

Reviewer: ```suggestion
description: "Control access to your dashboard with roles."
```

---

> Mintlify provides two dashboard access levels: Editor and Admin.

Reviewer: ```suggestion
Mintlify provides two dashboard access levels: Editor and Admin.

```
